README.md

```diff
@@ -1 +1,9 @@
 # genson-js
+
+**genson-js** is a user-friendly **JSON Schema** generator built in TypeScript/JavaScript.
+
+.. note::
```

Comment:
you can you `>` (instead of ..) to get a cool styling fro notes

Reply:
done